style.css

```diff
@@ -0,0 +1,352 @@
+    font-size: 1.68rem;
+    margin-right: 0.9rem;
+}
+.category-list{
+    margin: 0rem;
+    padding: 0rem;
+    justify-content: center;
+    font-size: 1.51rem;
+    font-family: Arial, Helvetica, sans-serif;
+    color: #494545;
+    height: 6rem;
+    border-bottom: 0.15rem solid #d6d6d6;
+    list-style: none;
+    letter-spacing: 0.07rem;
+    color: #6a6868;
+}
+
+.category-list__li{
+    height: 100%;
+    padding: 0rem 1.5rem;
+}
+.category-list__li:hover{
+    cursor: pointer;
+    background-color: #8080801c
+}
+
+.products-container{
+    width: 111.94rem;
+    margin: auto;
+    display: grid;
+    grid-template-columns: auto auto auto auto auto;
+    border: 0.15rem solid #d6d6d6;
+    border-top: 0rem;
+    border-radius: 0rem 0rem 0.45rem 0.45rem;
+}
+.products-container__category{
+    grid-column-start: 1;
+    grid-column-end: 2;
+    margin: 0rem;
+    padding: 0rem;
+}
+.products-container__product-image{
+    height: 4.48rem;
+    width: 4.48rem;
+    border-radius: 0.75rem;
+    border: none;
+    background-color: #f8f8f8;
+    display: flex;
+    justify-content: center;
+    align-items:flex-end;
+    margin-right: 1.5rem;
+}
+.products-container__product-image img{
+    height: 64%;
+}
+
+.products-container__category li{
+    padding: 0rem 1.2rem;
+    list-style: none;
+    width: 22.4rem;
+    height: 6rem;
+    font-size: 1.51rem;
+    border-bottom: 0.12rem solid #eee;
+    border-left: 0.6rem solid white;
+}
+
+.products-container__category li:hover:not(.products-container__active){
+    background-color: rgb(235, 255, 239);
+    border-left: 0.6rem solid rgb(235, 255, 239);
+    cursor: pointer;
+}
+.products-container__category .products-container__active{
+    background-color: rgb(235, 255, 239);
+    border-left: 0.6rem solid rgb(36 150 64);
+    cursor: pointer;
+}
+
+.products-container__items-container{
+    grid-column-start: 2;
+    grid-column-end: 5;
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+}
+.products-container__items-header{
+    justify-content: space-between;
+    grid-column-start: 1;
+    grid-column-end: 5;
+    padding: 1.65rem 1.5rem;
+} 
+.products-container__sort{
+    font-size: 1.34rem;
+    color: #666666;
+    margin-right: 0.75rem;
+}
+#sort-dropdown{
+    padding: 0.45rem;
+    color: green;
+    border: 0.06rem solid #cfc7c7;
+    border-radius: 0.3rem;
+}
+.products-container__buy-fresh-online{
+    font-size: 1.85rem;
+    font-weight: bold;
+}
+
+.products-container__items{
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+
+}
+.products-container__item{
+    padding: 1.5rem;
+    border: 0.07rem solid #d6d6d6;
+}
+.products-container__item-image{
+    position: relative;
+    height: 13rem;
+}
+
+.products-container__item-image img{
+    width: 70%;
+    margin-left: 2.58rem;
+}
+.products-container__discount{
+    width: 5rem;
+    background: rgb(37, 111, 239);
+    color: white;
+    padding: 0.22rem 0rem 0.22rem 0.22rem;
```

Comment:
Why right padding is zero?
How are you positioning this element? I don't see any left or top property

Reply:
I did not use the left or top property because the `products-container__discount` div is the first in the `products-container__item-image`. So, naturally it was taking the `left:0rem` and `top:0rem` values. Setting `position:absolute` allows the discount to overlay the image.